php/libraries/Database.class.inc

```diff
@@ -619,7 +619,7 @@ class Database
      */
     function selectRow($query, &$result)
     {
-        $this->select($query, $result);
+        $result = $this->pselect($query, []);
```

Comment:
`[]` => `array()`
Unless you want to start a new convention

Reply:
:(